homeassistant/components/sensor/brottsplatskartan.py

```diff
@@ -0,0 +1,136 @@
+"""Sensor for Brottsplatskartan."""
+from collections import defaultdict
+from datetime import timedelta
+import logging
+
+import voluptuous as vol
+
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+from homeassistant.const import (
+    ATTR_ATTRIBUTION,
+    ATTR_LATITUDE,
+    ATTR_LONGITUDE,
+    CONF_LATITUDE,
+    CONF_LONGITUDE,
+    CONF_NAME,
+)
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.entity import Entity
+
+REQUIREMENTS = ['brottsplatskartan==0.0.1']
+
+_LOGGER = logging.getLogger(__name__)
+
+CONF_AREA = 'area'
+DEFAULT_NAME = 'Brottsplatskartan'
+DOMAIN = 'brottsplatskartan'
+EVENT_INCIDENT = '{}_incident'.format(DOMAIN)
+SCAN_INTERVAL = timedelta(minutes=30)
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend(
+    {
+        vol.Inclusive(CONF_LATITUDE, 'coordinates'): cv.latitude,
+        vol.Inclusive(CONF_LONGITUDE, 'coordinates'): cv.longitude,
+        vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
+        vol.Optional(CONF_AREA, default=''): cv.string,
+    }
+)
+
+
+def setup_platform(hass, config, add_entities, discovery_info=None):
+    """Set up the Brottsplatskartan platform."""
+    import brottsplatskartan
+    import uuid
```

Comment:
uuid can be imported at the top of the module.

Reply:
Thanks for the feedback, Martin!

Importing uuid in the top of the module now.